lib/util/ssr.js

```diff
@@ -6,13 +6,45 @@
  */
 'use strict';
 
+let ssrEscapeFound = false;
```

Comment:
Having this `let` outside of `isSSREscape` makes it a global. This is a code smell to me – I'd recommend moving both `ssrEscapeFound` and `checkConditionalStatements` into `isSSREscape` instead.